### PR TITLE
Example on how to use the viewModel provided from layout in the phtml file

### PR DESCRIPTION
**Pull request by vasilii-b**
## Purpose of this pull request

This pull request (PR) shows how to use the view model in the `phtml` which was specified in the page layout.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-manage.html#layout_markup_add_to_elements

## Links to Magento source code
N/A

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 1]
```diff
@@ -484,9 +484,23 @@ class Class implements \Magento\Framework\View\Element\Block\ArgumentInterface
   {
 
   }
+
+  public function canShowAdditionalData() {
```
Comment (rogyar) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md]: Please, keep in mind PSR. It happens to me as well when working with javascript too much 😎

**Comment by rogyar** [comment]
Example from the codebase: 

https://github.com/rogyar/magento2/blob/a3d1c8a2f7787e549c7c024794b6f875acfcb550/app/code/Magento/Sales/view/adminhtml/templates/order/creditmemo/create/items.phtml#L11

**Comment by vasilii-b** [comment]
Thank you @rogyar,
Changes for the code style issue have been added.

**Comment by dobooth** [comment]
running tests

**Pull request merged**